modules/appeals_api/spec/factories/higher_level_reviews.rb

```diff
@@ -44,16 +67,13 @@
     end
   end
 
-  factory :minimal_higher_level_review, class: 'AppealsApi::HigherLevelReview' do
+  factory :minimal_higher_level_review_v2, class: 'AppealsApi::HigherLevelReview' do
     id { SecureRandom.uuid }
     auth_headers do
       JSON.parse File.read "#{::Rails.root}/modules/appeals_api/spec/fixtures/valid_200996_headers.json"
     end
     form_data do
-      JSON.parse File.read "#{::Rails.root}/modules/appeals_api/spec/fixtures/valid_200996_minimum.json"
-    end
-    trait :status_received do
-      status { 'received' }
+      JSON.parse File.read "#{::Rails.root}/modules/appeals_api/spec/fixtures/valid_200996_minimum_v2.json"
```

Comment:
Looks like there's a bug on minimum the minimum pdf where the single issue is being shown twice :( We'll have to fix that up, but I'll open a ticket for it so it doesn't stall this PR.

Reply:
ok!